client/my-sites/site-settings/navigation.jsx

```diff
@@ -30,11 +31,12 @@ export class SiteSettingsNavigation extends Component {
 			writing: translate( 'Writing', { context: 'settings screen' } ),
 			discussion: translate( 'Discussion', { context: 'settings screen' } ),
 			security: translate( 'Security', { context: 'settings screen' } ),
+			hosting: translate( 'Hosting Access', { context: 'settings screen' } ),
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.